src/main/httpCommandServer.ts

```diff
@@ -41,18 +52,35 @@ export default class HttpCommandServer {
     console.log('CLI server is now ready.');
   }
 
-  shutdown() {
-    this.server.close();
-  }
-
   protected handleRequest(request: http.IncomingMessage, response: http.ServerResponse) {
-    if (!this.basicAuth(request.headers.authorization ?? '')) {
-      response.writeHead(401, { 'Content-Type': 'text/plain' });
-    } else {
-      response.writeHead(200, { 'Content-Type': 'text/plain' });
-      response.write('Nothing to see here yet.');
+    try {
+      if (!this.basicAuth(request.headers.authorization ?? '')) {
+        response.writeHead(401, { 'Content-Type': 'text/plain' });
+
+        return;
+      }
+      const method = request.method;
```

Comment:
It's clearer to have this fallback, so that the response on missing looks better:
```suggestion
      const method = request.method ?? 'GET';
```
(So that when we get to `Unknown command ${ method } ${ commandName }`, `method` is always defined.)

Reply:
Done